okhttp/src/main/java/okhttp3/internal/platform/AndroidPlatform.java

```diff
@@ -50,8 +46,6 @@
   private final Class<?> sslParametersClass;
```

Comment:
"Android 2.3 or better" above

Reply:
Snuck in a fix here: https://github.com/square/okhttp/pull/4511